tests/ui/test_other_user.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+
+
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.utility.utils import get_ocp_version
+from ocs_ci.framework.testlib import ManageTest, ui
+from time import sleep
+from ocs_ci.utility.utils import exec_cmd
+
+logger = logging.getLogger(__name__)
+
+
+class OBCUI(PageNavigator):
```

Comment:
There is a class named the same name with some change [https://github.com/red-hat-storage/ocs-ci/blob/3e95ef866e80374d3d9fa165f878df7021a115f1/ocs_ci/ocs/ui/mcg_ui.py#L244](https://github.com/red-hat-storage/ocs-ci/blob/3e95ef866e80374d3d9fa165f878df7021a115f1/ocs_ci/ocs/ui/mcg_ui.py#L244).
I think here you  should only call the class from here. Please see if you can use the class that is already there in the link above or extend it if you need some other functionality.

Reply:
sure. I will extend this class for needed functionality